example_16/bringup/launch/diffbot.launch.py

```diff
@@ -14,158 +14,107 @@
+                "use_mock_hardware",
+                default_value="false",
+                description="Start robot with mock hardware mirroring command to its states.",
+            ),
+            DeclareLaunchArgument(
+                "fixed_frame_id",
+                default_value="odom",
+                description="Fixed frame id of the robot.",
+            ),
+            Node(
+                package="controller_manager",
+                executable="ros2_control_node",
+                parameters=[
+                    PathSubstitution(FindPackageShare("ros2_control_demo_example_16"))
+                    / "config"
+                    / "diffbot_chained_controllers.yaml"
+                ],
+                output="both",
+            ),
+            Node(
+                package="robot_state_publisher",
+                executable="robot_state_publisher",
+                output="both",
+                parameters=[
+                    {
+                        "robot_description": Command(
+                            [
+                                "xacro",
+                                " ",
+                                PathSubstitution(FindPackageShare("ros2_control_demo_example_16"))
+                                / "urdf"
+                                / "diffbot.urdf.xacro",
+                                " ",
+                                "use_mock_hardware:=",
+                                LaunchConfiguration("use_mock_hardware"),
+                            ]
+                        )
+                    }
+                ],
+            ),
+            Node(
+                package="rviz2",
+                executable="rviz2",
+                name="rviz2",
+                output="log",
+                arguments=[
+                    "-d",
+                    PathSubstitution(FindPackageShare("ros2_control_demo_description"))
+                    / "diffbot/rviz"
+                    / "diffbot.rviz",
+                    "-f",
+                    LaunchConfiguration("fixed_frame_id"),
+                ],
+                condition=IfCondition(LaunchConfiguration("gui")),
+            ),
+            Node(
+                package="controller_manager",
+                executable="spawner",
+                arguments=["joint_state_broadcaster"],
+            ),
+            Node(
+                package="controller_manager",
+                executable="spawner",
+                arguments=[
+                    "pid_controller_left_wheel_joint",
+                    "pid_controller_right_wheel_joint",
+                    "--param-file",
+                    PathSubstitution(FindPackageShare("ros2_control_demo_example_16"))
+                    / "config"
+                    / "diffbot_chained_controllers.yaml",
+                ],
+            ),
+            Node(
+                package="controller_manager",
+                executable="spawner",
+                arguments=[
```

Comment:
```suggestion
```
We have to launch the whole chain at once, so that the chained controllers are loaded in the correct order:

`    [ros2_control_node-1] [WARN] [1764196982.352153563] [controller_manager]: Unable to activate controller 'diffbot_base_controller' since the command interface 'pid_controller_left_wheel_joint/left_wheel_joint/velocity' is not available.`